src/compose.ts

```diff
@@ -694,7 +692,7 @@ export const composeHandler = ({
 			}`
 		} else {
 			fnLiteral += `if(c.qi !== -1) {
-				let url = '&' + decodeURIComponent(c.url.slice(c.qi + 1))
+				let url = '&' + decodeURIComponent(c.url.slice(c.qi + 1).replaceAll('+', ' '))
```

Comment:
We should replace `+` with ` ` before calling `decodeURIComponent`.
Otherwise, we won't know whether a `+` comes from a space (and we want to replace it with space) or comes from a `%2B` (and we want to keep it).